pype/plugins/global/publish/extract_review.py

```diff
@@ -453,6 +453,7 @@ def ffmpeg_full_args(
         if audio_filters:
             all_args.append("-filter:a {}".format(",".join(audio_filters)))
 
+        all_args.append('-vf pad="width=ceil(iw/2)*2:height=ceil(ih/2)*2"')
```

Comment:
This will break what is happening in `def rescaling_filters`.

Reply:
any idea how to fix it the way it will not break those filters?